### ISSUE TITLE
Lost all friends avatar after removing password from my profile.......

### ISSUE THREAD

**Issue by DjDiabolik** [qTox/qTox#4172]
##### Brief Description

OS: Windows 10 Home X64 
qTox version: 1.7.0
Commit hash:  12c21b21d2d62a27b0002c1add7e4d7faa0d018d
toxcore:  0.1.5
Qt:  5.5.1 

Reproducible: Always 

##### Steps to reproduce

1.  I have removed password from my profiles for obtain back the autologin works
2. On friends list now all the image picture it's not more present and no more loaded.
3. Also my friends can see my profile picture only when chat it's opened and not in our friendly list
4. I have also tryed to reset a new password for my profile. Close QTox. Re-open it but nothings changed... the image picture it's not more appears.. it's normal ?

**Comment by DjDiabolik** [comment]
mmmmmm.. i also can add.. i see a 1.8.1 it's present.. because no propose to me to upgrade ?

**Comment by DjDiabolik** [comment]
I have an example................ look there's a chat windows whit a my friends.

Look his avatar..........it's present on Chat windows but now in windows list.......  It's normal ? It's a bug ?

Link to watch the screenshot:
http://imgur.com/iJG6gG1

**Comment by TheNain38** [comment]
@tux3 If I remember correctly, when the profile is encrypted, the avatars are saved inside the database, but when it's not encrypted, the avatars are saved inside the: "avatars" folder. So, if your profile was encrypted and then, later you decrypt it, all the avatars that were saved inside the database, won't be saved inside it anymore, so they will be lost, and if you re-encrypt the profile, because the database is now cleared of any avatars, they still won't be there after re-encrypting

**Comment by tux3** [comment]
@TheNain38 Yes, I suspect something like that. But I though there was already code for decrypting the avatars when changing or removing the password, so there must be a problem with that code.

**Comment by sudden6** [comment]
If I read the code correctly, the avatars are never saved in the database.

**Comment by DjDiabolik** [comment]
meanwhile i have try to see this path:
C:\Users\MyWindowsUser\AppData\Roaming\Tox\avatars

Inside this folder i can see it's present 5 .png file... one it's my avatar... one it's the avatar also you can see on my screenshot above and 3 other .png if i try to open whit irfanview sat it's corrupted and say it's impossible to read the header of files.
Apparently the names of two .png i can open whit irfanviews it's the ID of nickname owners..... it's possible i can delete this 3 other like currupted files can be replaced next time i launch qtox clients ?

**Comment by sudden6** [comment]
@DjDiabolik yes you can try to delete them, they will be redownloaded from your friends when both of you are online at the same time.

I think they show as corrupted, because they got encrypted, but never decrypted apparently.
